apps/infrastructure/src/acm.tf

```diff
@@ -11,4 +12,34 @@ module "acm" {
   wait_for_validation = true
   validation_method   = "DNS"
   dns_ttl             = 3600
+}
+
+## Certificate HTTPS for Cloudfront Websites
```

Comment:
I don't think this comment is related to the `provider` block below

Reply:
comment moved with next commit